src/Playwright/Core/Page.cs

```diff
@@ -286,8 +286,9 @@ public Task EmulateMediaAsync(PageEmulateMediaOptions options = default)
             var args = new Dictionary<string, object>
             {
                 ["media"] = options?.Media,
-                ["colorScheme"] = options?.ColorScheme,
-                ["reducedMotion"] = options?.ReducedMotion,
+                ["colorScheme"] = options?.ColorScheme ?? ColorScheme.Null,
```

Comment:
I think there was a semantics difference between the Null and no value that is regressing.

Reply:
Hm, I fixed the tests to be aligned with upstream. Unless I'm missing something else?